File: sample-game/src/main/java/com/github/ummo93/Launcher.java
package com.github.ummo93;

import com.github.ummo93.config.ModuleConfig;
import com.github.ummo93.framework.JayginApplication;

public class Launcher {
    public static void main(String[] args) {
        var game = new JayginApplication(new ModuleConfig());
        game.start();
    }
}

File: sample-game/src/main/kotlin/Launcher.kt
package com.github.ummo93

import com.github.ummo93.config.ModuleConfig
import com.github.ummo93.framework.JayginApplication

object Launcher {
    @JvmStatic
    fun main(args: Array<String>) {
        val game = JayginApplication(ModuleConfig())
        game.start()
    }
}